react/src/loginForm.js

```diff
@@ -0,0 +1,76 @@
+import { Form, Input, Button, Checkbox } from 'antd';
+import React from 'react';
+
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.submitFormForLogin = this.submitFormForLogin.bind(this);
```

Comment:
I asked you to make corrections in a bind. Can you refer to the documentation to "bind" usages and make appropriate changes?